reactor-netty-http/src/main/java/reactor/netty/http/server/HttpServerRoutes.java

```diff
@@ -262,6 +263,25 @@ default HttpServerRoutes put(String path,
 	HttpServerRoutes route(Predicate<? super HttpServerRequest> condition,
 			BiFunction<? super HttpServerRequest, ? super HttpServerResponse, ? extends Publisher<Void>> handler);
 
+	/**
+	 * Use the provided {@link java.util.Comparator} to sort routes, rather than using configured order.Routes that were
+	 * already configured are also impacted by this change and will be sorted according to the comparator.You can revert
+	 * to using the declaration order by calling the {@link #noComparator()} method (which is the default).
+	 *
+	 * @param comparator a HttpRouteHandlerMetadata comparator.
+	 * @return this {@link HttpServerRoutes}
+	 * @since 1.0.7
+	 */
+	HttpServerRoutes comparator(Comparator<HttpRouteHandlerMetadata> comparator);
+
+	/**
+	 * Do not sort routes but use the order which the routes were configured.
```

Comment:
missing `in` in `user the order IN which`. Also, we talk about the `declaration` order above, so for consistency `configured` should be `declared`.

Reply:
Removes any previously applied `{@link java.util.Comparator}` customization using `{@link HttpServerRoutes#comparator(Comparator)}`, and use the order in which the routes were configured.

Reply:
@ctlove0523 Yep 👍

Reply:
@simonbasle PTAL